recipes/default.rb

```diff
@@ -15,52 +15,201 @@
+  package %w(
+    automake
+    ImageMagick-devel
+    libffi-devel
+    libtool
+    libyaml-devel
+    openssl-devel
+    postgresql-devel
+    readline-devel
+    sqlite-devel
+    zlib-devel
+  )
+
+  package 'osl-app packages' do
+    package_name osl_app_packages
+  end
+
+  package 'python-psycopg2'
+
+  # Keep systemd services private from non-root users
+  directory '/etc/systemd/system' do
+    mode '750'
+  end
+
+  # rewind the sudoers template to support sudoers_d
+  temp = resources(template: '/etc/sudoers')
+  temp.variables['include_sudoers_d'] = true
+
+  build_essential 'install tools'
+
+  include_recipe 'osl-nodejs'
+
 end
-package 'python-psycopg2' do
-  package_name 'python2-psycopg2' if node['platform_version'].to_i >= 8
+
+osl_firewall_port 'unicorn' do
+  osl_only true
 end
 
-# Keep systemd services private from non-root users
-directory '/etc/systemd/system' do
-  mode '750'
+# Enable live-restore to keep containers running when docker restarts
+node.override['osl-docker']['service'] = { misc_opts: '--live-restore' }
+
+include_recipe 'osl-docker'
+
+# Docker containers - app2
+directory '/data/docker/redmine.replicant.us' do
+  recursive true
 end
 
-# rewind the sudoers template to support sudoers_d
-temp = resources(template: '/etc/sudoers')
-temp.variables['include_sudoers_d'] = true
+docker_image 'osuosl/redmine-replicant' do
+  tag '4.2.3-2022.01.14.1907'
+  action :pull
+end
 
-build_essential 'install tools'
+replicant_dbcreds = data_bag_item('replicant_redmine', 'mysql_creds')
+replicant_dbcreds['db_hostname'] = node['ipaddress'] if node['kitchen']
 
-include_recipe 'git'
-include_recipe 'osl-nodejs'
+docker_container 'redmine.replicant.us' do
+  repo 'osuosl/redmine-replicant'
+  tag '4.2.3-2022.01.14.1907'
+  port '8090:3000'
+  restart_policy 'always'
+  volumes ['/data/docker/redmine.replicant.us:/usr/src/redmine/files']
+  env [
+    "REDMINE_DB_MYSQL=#{replicant_dbcreds['db_hostname']}",
+    "REDMINE_DB_DATABASE=#{replicant_dbcreds['db_db']}",
+    "REDMINE_DB_USERNAME=#{replicant_dbcreds['db_user']}",
+    "REDMINE_DB_PASSWORD=#{replicant_dbcreds['db_passwd']}",
+    'REDMINE_PLUGINS_MIGRATE=1',
+  ]
+  sensitive true
+end
 
-osl_firewall_port 'unicorn' do
-  osl_only true
+git '/var/lib/formsender' do
+  repository 'https://github.com/osuosl/formsender.git'
+  revision 'master'
+  notifies :build, 'docker_image[formsender]', :immediately
+  notifies :redeploy, 'docker_container[formsender]'
+end
+
+docker_image 'formsender' do
+  tag 'latest'
+  source '/var/lib/formsender'
+  action :nothing
+end
+
+formsender_env = data_bag_item('osl-app', 'formsender')
+
+docker_container 'formsender' do
+  repo 'formsender'
+  tag 'latest'
+  port '8085:5000'
+  restart_policy 'always'
+  env [
+    "TOKEN=#{formsender_env['token']}",
+    "RT_TOKEN=#{formsender_env['rt_token']}",
+    "RECAPTCHA_SECRET=#{formsender_env['recaptcha_secret']}",
+  ]
+  sensitive true
+end
+
+# Docker containers - app3
```

Comment:
None of this should be included in the default recipe.

Reply:
Ohh, I think I misunderstood what you wrote earlier. Should I only be moving the `include_recipe 'osl-docker'` and the `node.override['osl-docker']['service'] = { misc_opts: '--live-restore' }` lines and not the actual container creation blocks? @ramereth 
> * Move osl-docker references currently in app2/app3 to the default recipe and run on both EL7 and EL8

Reply:
yes! Sorry if I confused you earlier.

Reply:
No worries! I will get that fixed.

Reply:
Done!